src/kernels/execution/cellExecution.ts

```diff
@@ -301,9 +309,19 @@ export class CellExecution implements IDisposable {
         const props = { notebook: this.controller.notebookType === JupyterNotebookView };
         if (!CellExecution.sentExecuteCellTelemetry) {
             CellExecution.sentExecuteCellTelemetry = true;
-            sendTelemetryEvent(Telemetry.ExecuteCellPerceivedCold, this.stopWatchForTelemetry.elapsedTime, props);
+            sendKernelTelemetryEvent(
```

Comment:
Track this per resource, so we have more information about the startup times, e.g. per kernal language, remote vs local, etc.